src/framework/preferences/js/PrefsEditor.js

```diff
@@ -391,13 +391,17 @@ var fluid_2_0 = fluid_2_0 || {};
             fluid.model.diff(userSelections.preferences, fluid.get(that.initialModel, ["preferences"]), stats);
 
             fluid.each(stats.changeMap, function (state, path) {
-                fluid.set(changedSelections, ["preferences", path], userSelections.preferences[path]);
+                fluid.set(changedPrefs, path, userSelections.preferences[path]);
```

Comment:
One reason that the save function saves the entire model is because of the special handling of the "preferences" path that only changed preferences are saved instead of a direct save of all preferences. This allows the ability when the initial model gets changed later for unchanged preferences, the new initial values can still be merged correctly with the saved prefs. This special handling is currently transparent to integrators. Allowing users to pass in the path parameter would increase the complexity of maintaining this transparency, in terms of figuring out where "preferences" are since they need a special handling, merging the value of the given path into the saved model, the possibility of parts of the saved model and the component model could be out of sync etc. Also, as far as I know, we don't have an use case so far that requires to save a specific path of the prefsEditor model.

I think this topic worths more discussions and likely a separate JIRA.